README.md

```diff
@@ -21,7 +21,8 @@ campaign assessors to simplify the process of managing GoPhish campaigns.
 * `gophish-complete` - Completes a campaign in GoPhish and/or outputs a
   GoPhish campaign summary.
 * `gophish-export` - Exports all the data from an assessment within GoPhish
```

Comment:
Please make an issue to replace all instances of `GoPhish` with `Gophish` in this project. We've slowly been making the change in other projects as we do updates and I don't want to scope creep this PR.

Reply:
See [here](https://github.com/cisagov/gophish-tools/issues/35) for the new issue